apps/address-service/domains/common/utils/services/suggest/SuggestionKeystoneApp.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express')
+
+const get = require('lodash/get')
+const { SuggestionProviderDetector } = require('@address-service/domains/common/utils/services/suggest/SuggestionProviderDetector')
+
+class SuggestionKeystoneApp {
+    /**
+     * @param params
+     * @returns {Express}
+     */
+    prepareMiddleware (params) {
+        const app = express()
+        const providerDetector = new SuggestionProviderDetector()
+
+        function setNoCache (req, res, next) {
+            res.set('Pragma', 'no-cache')
+            res.set('Cache-Control', 'no-cache, no-store')
+            next()
+        }
+
+        app.get('/suggest', setNoCache, async (req, res, next) => {
+            /**
+             * Using to detect a proper suggestion provider
+             * @type {string}
+             */
+            const geo = String(get(req, ['query', 'geo'], null))
```

Comment:
country?

Reply:
I thought about `country` and `locale` and others. It looks like `geo` is the universal name for the variable that influences to detection of a provider.

Reply:
`geo` looks more like geoposiiton of one's that sent the request. But what if a person is temporary in another country?

Reply:
Name `geo` doesn't fit a bit as like the other ones :)
I'd like to keep this name until particular reasons for changing appear.